src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -158,7 +158,29 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers {
 
   }
 
-  it should "clone all entities from a workspace containing cycles" in withDefaultTestDatabase { 
+  it should "update an entity's attributes many times concurrently" in withDefaultTestDatabase {
```

Comment:
should we worry about creating the same entity concurrently?  for instance if someone uploaded the same tsv twice in a row (e.g. saw an error and retried)